src/Opdex.Platform.Infrastructure/Data/DbContext.cs

```diff
@@ -13,6 +15,11 @@ namespace Opdex.Platform.Infrastructure.Data;
 // Ref: https://stackoverflow.com/questions/538060/proper-use-of-the-idisposable-interface
 public class DbContext : IDbContext
 {
+    private readonly AsyncRetryPolicy _retryPolicy = Policy
+        .Handle<MySqlException>(ex => ex.IsTransient)
+        .Or<TimeoutException>()
+        .WaitAndRetryAsync(3, x => TimeSpan.FromSeconds(x));
+
```

Comment:
Wooops, my bad I pushed the Dapper changes to this branch!
Anyway, it's a small change. The developers of the MySql extension were nice enough to provide a property that indicates if an exception may be transient, so it was a lot easier than I thought it would be.